supervision/tracker/byte_tracker/core.py

```diff
@@ -286,20 +283,22 @@ def callback(frame: np.ndarray, index: int) -> np.ndarray:
             iou_costs = 1 - ious
 
             matches, _, _ = matching.linear_assignment(iou_costs, 0.5)
-            for i, idet, itrack in enumerate(matches):
-                if i == 0:
-                    final_detections = detections[[idet]]
-                    final_detections.tracker_id[0] = int(tracks[itrack].track_id)
-                else:
-                    current_detection = detections[[idet]]
-                    current_detection.tracker_id[0] = int(tracks[itrack].track_id)
-                    final_detections = Detections.merge(
-                        [final_detections, current_detection]
-                    )
+            detections.tracker_id = np.full(len(detections), -1, dtype=int)
+            for i_detection, i_track in matches:
+                detections.tracker_id[i_detection] = int(tracks[i_track].track_id)
+
+            invalid_tracks = [
```

Comment:
@rolson24 I assume that after https://github.com/roboflow/supervision/pull/1062 we can change this into `return detections[detections.tracker_id != =1]` ?